airflow/providers/cncf/kubernetes/hooks/kubernetes_hook.py

```diff
@@ -0,0 +1,77 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```

Comment:
We should rename the file to `airflow/providers/cncf/kubernetes/hooks/kubernetes.py` like the other hooks